lua/entities/gmod_wire_egp/init.lua

```diff
@@ -48,4 +49,4 @@ function ENT:GetEGPOwner()
 	return false
 end
 
-function ENT:UpdateTransmitState() return TRANSMIT_ALWAYS end
+function ENT:UpdateTransmitState() return TRANSMIT_PVS end
```

Comment:
This shouldn't be changed. The entity should always exist for clients in order to stay in sync.